assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm

```diff
@@ -44,7 +44,8 @@
 			<span class="group_icon" title="$tlang.getString('gen.groupassignment')" alt="$tlang.getString('gen.groupassignment')"></span>
 		#else
 			<span class="user_icon" title="$tlang.getString('gen.userassignment')" alt="$tlang.getString('gen.userassignment')"></span>
-		#end <span class="highlight">  - $!tlang.getString('gen.subm2') </span>
+		#end
+		  - $!tlang.getString('gen.subm2')
```

Comment:
I think you can drop the `!` as `tlang` should always be in the context unless something has gone really badly wrong.